jupyter_server_proxy/__init__.py

```diff
@@ -17,6 +25,12 @@ def _jupyter_nbextension_paths():
         "require": "jupyter_server_proxy/tree"
     }]
 
+def _jupyter_labextension_paths():
+    return [{
+        "src": "labextension",
+        "dest": data["name"]
```

Comment:
This is not something we should be changing, so I figure we don't need to have logic to import this name.

```suggestion
        "dest": "@jupyterhub/jupyter-server-proxy",
```